### ISSUE TITLE
Invalid format strings in pywbem_mock/_resolvermixin.py

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1817]
In pywbem_mock/_resolvermixin.py, some format strings use vertical bar instead of exclamation mark:

```
_format("Qualifier {0|A} in new_class {1|A} in "
_format("Qualifier {0|A} in new_class {1|A} override type "
_format("Qualifier {0|A} in new class {1|A} scope {2|A} "
_format("New class {0|A} derived from superclass {1!A} "
```

**Comment by andy-maier** [comment]
In addition, the arguments are specified as a tuple rather than just arguments.
In addition, use of incorrect attribute `classnam`.

**Comment by andy-maier** [comment]
Fixed by PR #1818 in master, and PR #1819 rolls the fix back into 0.14.5.
